CoCSharp.Client/RemoteServer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Net;
+using System.Net.Sockets;
+
+namespace CoCSharp.Client
+{
+    class RemoteServer
```

Comment:
yeah look at the commit title which added it: wip stuff i needed to add for travis-ci to work